.gitignore

```diff
@@ -1,9 +1,8 @@
-.DS_Store
```

Comment:
Is this a duplicate or something?

Reply:
yep - it appears again [at the end of the file](https://github.com/SupertechLabs/SupermarQ/blob/2e2c049fa8c09fe941a9d8514c4b763f5c8cfdbe/.gitignore#L153)

Reply:
:+1: